src/components/UploadNoStyle/UploadNoStyle.tsx

```diff
@@ -0,0 +1,86 @@
+import { t } from '@lingui/macro'
+import { Upload, UploadProps } from 'antd'
+import { RcFile } from 'antd/lib/upload'
+import { FormItemInput } from 'models/formItemInput'
+import { ReactNode, useCallback, useState } from 'react'
+import { emitErrorNotification } from 'utils/notifications'
+
+interface UploadNoStyleProps
+  extends FormItemInput<string | undefined>,
+    Omit<
+      UploadProps,
+      'onChange' | 'showUploadList' | 'children' | 'customRequest'
+    > {
+  sizeLimit?: number
+  supportedFileTypes?: Set<'image/jpeg' | 'image/png' | 'image/gif'>
+  customRequest?: (req: File | Blob | string) => Promise<string> | string
+  children?: (props: {
+    isUploading: boolean
+    uploadUrl: string | undefined
+    undo: VoidFunction
+  }) => ReactNode
+}
+
+export const UploadNoStyle = (props: UploadNoStyleProps) => {
+  const [_uploadUrl, _setUploadUrl] = useState<string>()
+  const uploadUrl = props.value ?? _uploadUrl
+  const setUploadUrl = props.onChange ?? _setUploadUrl
+
+  const [isUploading, setIsUploading] = useState<boolean>(false)
```

Comment:
**nitpick** just as an org thing, i tend to like grouping `useStates` together. NBD. I wonder if we could lint for it.

Reply:
yeah me too - will fix